static/app/views/settings/projectMetrics/customMetricsTable.tsx

```diff
@@ -153,11 +152,6 @@ function MetricsTable({metrics, isLoading, query, project}: MetricsTableProps) {
     <MetricsPanelTable
       headers={[
         t('Metric'),
-        <Cell right key="cardinality">
-          <IconArrow size="xs" direction="down" />
```

Comment:
They were sorted by cardinality. That probably still makes sense if something is exceeding the limit, to have it on top. Otherwise, sorting by name is probably more helpful now and appears less random to the user.